config/locales/fr.yml

```
@@ -225,11 +225,14 @@ fr:
     update_info_successfully: Informations de mise à jour récupérées
   agents:
     add_agent: Nouvel agent ajouté
+    edit_agent: Modifier l'agent 
```

YAMLlint (1.35.1)

[error] trailing spaces

(trailing-spaces)